src/build123d/_version.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
According to https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package, there should be a `__version__ = "0.0.1"` here. Could this be why the builds are failing?

Reply:
added __version__ = "0.0.1" as requested, and imported in `init.py`